app/controllers/subscriptions_controller.rb

```diff
@@ -46,4 +48,8 @@ def daily_subscription_params
   def search_criteria
     params.require(:search_criteria).permit(*VacancyFilters::AVAILABLE_FILTERS)
   end
+
+  def check_feature_flag
+    raise(ActionController::RoutingError, 'Not Found') unless EmailAlertsFeature.enabled?
```

Comment:
We have a `not_found` method in ApplicationController you might be able to use to raise a 404 consistently, and with styling.

Reply:
Ah, nice, I did look for something like this, but only searching the codebase for `404`, should've checked the ApplicationController